docs/getting-started-on-linux.md

```diff
@@ -0,0 +1,109 @@
+## Getting Started
+
+Follow the development machine setup on each desktop you'd like to use.
+
+### Development Machine Setup
+
+Clone the repository locally from terminal:
+
+```markdown
+> cd (to working directory of your choosing)
+> git clone https://github.com/microsoft/do-client
+```
+
+Run the appropriate bootstrapper depending on development machine platform:
+
+```markdown
+> cd build/bootstrap
+> ./bootstrap-ubuntu-18.04.sh
+```
+
+### Building DO client components 
+**NOTICE:**  
+**If you are modifying this project and distributing your own custom build, please modify the DO_BUILDER_IDENTIFIER cmake variable located in https://github.com/microsoft/do-client/blob/main/CMakeLists.txt**
```

Comment:
>https://github.com/microsoft/do-client/blob/main/CMakeLists.txt [](start = 142, length = 63)

Turn this into a proper link [CMakeLists.txt](https://github.com/microsoft/do-client/blob/main/CMakeLists.txt)